### PR TITLE
Wordpad translation

### PR DESCRIPTION
**Pull request by Andrej123456789**
## Purpose
Translate Wordpad to Craotian

JIRA issue: no issue

## Proposed changes

-  Update rsrc.rc
-  Upload hr-HR.rc

## TODO
No tasks to do, I builded and tested

### CODE REVIEW

**Comment by Andrej123456789** [comment]
Hrvatski
Picture of translation

File: base/applications/wordpad/lang/hr-HR.rc [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    EDITTEXT IDC_PARA_FIRST, 75, 56, 65, 13
+    LTEXT "Poravnanje", -1, 15, 87, 60, 13
+    COMBOBOX IDC_PARA_ALIGN, 75, 85, 65, 60, CBS_DROPDOWNLIST
+    DEFPUSHBUTTON "U redu", IDOK, 160, 15, 50, 15
+    PUSHBUTTON "Odustani", IDCANCEL, 160, 33, 50, 15
+END
+
+IDD_TABSTOPS DIALOGEX 30, 20, 200, 110
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
+CAPTION "Kartice"
```
Comment (HBelusca) [inline base/applications/wordpad/lang/hr-HR.rc:231]: What's the difference between "Kartice" & "Tabulatori" ?

File: base/applications/wordpad/lang/hr-HR.rc [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        MENUITEM "Bijela",      ID_COLOR_WHITE
+        MENUITEM "Automatski",  ID_COLOR_AUTOMATIC
+    END
+END
+
+STRINGTABLE
+BEGIN
+    STRING_ALL_FILES,              "Svi dokumenti (*.*)"
+    STRING_TEXT_FILES_TXT,         "Tekstni dokumenti (*.txt)"
+    STRING_TEXT_FILES_UNICODE_TXT, "Unicode tekstualni dokument (*.txt)"
```
Comment (HBelusca) [inline base/applications/wordpad/lang/hr-HR.rc:132]: What's the difference between "tekstualni dokument" and "tekstni dokument" (that seems to be used everywhere else)?

File: base/applications/wordpad/lang/hr-HR.rc [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    END
+    POPUP "&Uredi"
+    BEGIN
+        MENUITEM "&Poništi\tCtrl+Z",               ID_EDIT_UNDO
+        MENUITEM "&Vrati\tCtrl+Y",               ID_EDIT_REDO
+        MENUITEM SEPARATOR
+        MENUITEM "&Izreži\tCtrl+X",                ID_EDIT_CUT
+        MENUITEM "&Kopiraj\tCtrl+C",               ID_EDIT_COPY
+        MENUITEM "&Zalijepi\tCtrl+V",              ID_EDIT_PASTE
+        MENUITEM "I&zbriši\tDel",                 ID_EDIT_CLEAR
```
[the remark this answers is not in the document]
Comment (Andrej123456789) [reply base/applications/wordpad/lang/hr-HR.rc:41]: I'll fix it

File: base/applications/wordpad/lang/hr-HR.rc [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+        MENUITEM "&Označi sve\tCtrl+A",         ID_EDIT_SELECTALL
+        MENUITEM SEPARATOR
+        MENUITEM "&Pronađi...\tCtrl+F",            ID_FIND
```
[the remark this answers is not in the document]
Comment (Andrej123456789) [reply base/applications/wordpad/lang/hr-HR.rc:44]: I'll fix it

File: base/applications/wordpad/lang/hr-HR.rc [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+        MENUITEM "Pronađi &slijedeće\tF3",              ID_FIND_NEXT
+        MENUITEM "&Zamijeni...\tCtrl+H",         ID_REPLACE
```
[the remark this answers is not in the document]
Comment (Andrej123456789) [reply base/applications/wordpad/lang/hr-HR.rc:46]: I'll fix it

File: base/applications/wordpad/lang/hr-HR.rc [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+    POPUP "&Ubaci"
+    BEGIN
+        MENUITEM "&Datum i vrijeme...",  ID_DATETIME
+    END
+    POPUP "&Format"
+    BEGIN
+        MENUITEM "&Font...",                     ID_FONTSETTINGS
+        MENUITEM "&Grafičke oznake",               ID_BULLET
+        MENUITEM "&Paragraf...",                ID_PARAFORMAT
+        MENUITEM "&Kartice...",                     ID_TABSTOPS
```
[the remark this answers is not in the document]
Comment (Andrej123456789) [reply base/applications/wordpad/lang/hr-HR.rc:78]: Yes

File: base/applications/wordpad/lang/hr-HR.rc (hunk 2 above)
Comment (Andrej123456789) [reply base/applications/wordpad/lang/hr-HR.rc:132]: No difference, I will fix it

**Pull request merged**